contact/style.scss

```
@@ -0,0 +1,143 @@
+body{
+  background-color: #FFFFFF;
```

```suggestion
  background-color: #FFF;
```
同じ文字が6桁の場合、3桁表示にできます。

---

問題ありません